oepo.owl

```diff
@@ -356,23 +386,12 @@ The Object Ontology is licensed by CC 4.0 (http://creativecommons.org/licenses/b
     
 
 
-    <!-- http://www.phenome-fppn.fr/vocabulary/2017#RGBequivalentWavelength -->
-
-    <owl:DatatypeProperty rdf:about="http://www.phenome-fppn.fr/vocabulary/2017#RGBequivalentWavelength">
-        <rdfs:subPropertyOf rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#sensorProperty"/>
-        <rdfs:domain rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#VNIRCamera"/>
-        <rdfs:label xml:lang="en">RGB equivalent wavelength (central wavelength, nm)</rdfs:label>
-        <rdfs:label xml:lang="fr">bandes équivalentes RGB (longueur d&apos;onde centrale, nm)</rdfs:label>
-    </owl:DatatypeProperty>
-    
-
+    <!-- http://www.phenome-fppn.fr/vocabulary/2017#aperture -->
 
-    <!-- http://www.phenome-fppn.fr/vocabulary/2017#apperture -->
-
-    <owl:DatatypeProperty rdf:about="http://www.phenome-fppn.fr/vocabulary/2017#apperture">
-        <rdfs:subPropertyOf rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#sensorProperty"/>
-        <rdfs:domain rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#Optic"/>
-        <rdfs:label xml:lang="en">apperture (fnumber)</rdfs:label>
+    <owl:DatatypeProperty rdf:about="http://www.phenome-fppn.fr/vocabulary/2017#aperture">
```

Comment:
To discuss: for properties call hasAperture instead of aperture wich could be confound with concept.